### ISSUE TITLE
No calendar data pulling through for eastern-cape-kareedouw.ics

### ISSUE THREAD

**Issue by KheffiPL** [beyarkay/eskom-calendar#207]
Either pre-viewing the calender (from iPhone), or subscribing to the .ics yields no events...
Tested while already having another calendar successfully subscribed, and adding and removing others to test.

https://github.com/beyarkay/eskom-calendar/releases/download/latest/eastern-cape-kareedouw.ics

**Comment by beyarkay** [comment]
Hmm, that's interesting. I'll have a look tonight and see if I can figure out the problem

**Comment by beyarkay** [comment]
So I've had a look and can confirm that `eskom-calendar` currently doesn't see any loadshedding at all for kareedouw: 
```
$ curl -sL https://github.com/beyarkay/eskom-calendar/releases/download/latest/machine_friendly.csv \
    | rg "eastern-cape-kareedouw"
// no content
```

despite there very definitely being loadshedding now. 

The problem is that the [CSV file that defines the schedule for Kareedouw](https://github.com/beyarkay/eskom-calendar/blob/main/generated/eastern-cape-kareedouw.csv) is empty, basically because 5c1aff2113f049d4aee92821af1010ffcc075db5 removed the entire file.

This shouldn't be too tricky to resolve, but I'll only be able to get to it in the morning. In the mean time, if you've subscribed to the [eastern-cape-kareedouw.ics](https://github.com/beyarkay/eskom-calendar/releases/download/latest/eastern-cape-kareedouw.ics) digital calendar feed then the loadshedding schedule will automatically appear once I've sorted out the problem.


Thanks for opening the issue! it really helps us out. Can I ask where you heard about `eskom-calendar` from?

**Comment by beyarkay** [comment]
Strangely enough, it seems like `kareedouw` was literally the only calendar to be deleted as a result of 5c1aff2. This can be checked by getting a word-count `wc` of the lines `-l` in all the files in the `generated` directory (`generated/*`):

```
$ wc -l generated/* | sort | head
       1 generated/eastern-cape-kareedouw.csv
     611 generated/gauteng-emfuleni.pdf
     620 generated/gauteng-ekurhuleni-block-7.csv
     624 generated/gauteng-ekurhuleni-block-11.csv
     624 generated/gauteng-ekurhuleni-block-15.csv
     624 generated/gauteng-ekurhuleni-block-3.csv
     624 generated/gauteng-ekurhuleni-block-4.csv
     624 generated/gauteng-ekurhuleni-block-5.csv
     624 generated/gauteng-ekurhuleni-block-6.csv
     625 generated/gauteng-ekurhuleni-block-1.csv
```

Kareedouw has only one line (the default header line) but all other files have a few hundred. Not sure why this is the case.

**Comment by beyarkay** [comment]
Turns out the official eskom loadshedding schedule for Kareedouw has *nothing inside of it*: https://www.eskom.co.za/distribution/wp-content/uploads/2021/06/Kareedouw-1.pdf which was why `eskom-calendar` was showing there to be no loadshedding.

@KheffiPL do you get your loadshedding schedule from EskomSePush? or is there another website that you use? It's pretty easy to add the schedule in, but I like to find something official and a primary source rather than going off of word of mouth.

**Comment by KheffiPL** [comment]
“@KheffiPL do you get your loadshedding schedule from EskomSePush?”

Yes sir, is available on EskomSePush. Have it under Eersterivier, Eskom Direct, Kou-Kamma, Eastern Cape.

**Comment by beyarkay** [comment]
Hi @KheffiPL , This should be solved now by 13e10bcb87bc2844c64e4ddf9d238e46ccd865fa. You can search for "kareedouw" in [eskomcalendar.co.za](https://eskomcalendar.co.za/ec) and it will show you the correct schedule.

I hope that helps! If you like the project, leaving a "star" (there's a button on [the main page](https://github.com/beyarkay/eskom-calendar)) would be much appreciated, as this helps more people discover eskom-calendar.
